aavealarm/network.tsx

```diff
@@ -103,6 +104,10 @@ function getAaveMarket(
       market = markets.AaveV3Metis;
       chainId = ChainId.metis_andromeda;
       break;
+    case chain == Chain.BASE && aaveVersion == 3:
+        market = markets.AaveV3Base;
+        chainId = ChainId.base;
```

Comment:
When I tried it locally, I saw an error that `ChainId.base` doesn't exist. Did I do anything wrong?

Reply:
Got this on my side:

https://github.com/nebolax/aavealarm/assets/11545946/4dd9d815-9398-4e09-9bbf-b29843683364

Reply:
And I suppose `@aave/contract-helpers` should be updated in `package.json` too.

Reply:
Package have been updated